setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    # TODO: Decide name for package
+    name="TO BE DECIDED",
```

Comment:
"UVic Software Engineering Course Scheduler" or "Course Scheduler" if it must match the package name